AmazonChimeSDK/AmazonChimeSDKTests/audiovideo/data/transcript/TranscriptAlternativeTests.swift

```diff
@@ -13,10 +13,11 @@ class TranscriptAlternativeTests: XCTestCase {
     let transcript = "test"
 
     func testTranscriptAlternativeShouldInitialize() {
-        let transcriptAlternative = TranscriptAlternative(items: [], transcript: transcript)
+        let transcriptAlternative = TranscriptAlternative(items: [], entities: [], transcript: transcript)
```

Comment:
If entities is optional we need a test case covering that as well. 

This is a silly test -- just testing the constructor. I don't think there's any value here. This should be implicitly tested through other test cases.

Reply:
Yeah, we would have test coverage for optional param.

Reply:
added this here to get the tests to pass. will expand tests.